MAINTAINERS_GUIDE.md

```diff
@@ -14,6 +14,9 @@ Maintainers should create new issues as needed.
 ### Review Pull Requests
 
 * Maintainers should regularly review the backlog of pull requests. Pull requests only require one maintainer to approve. The maintainer reviewing the PRs should be a subject matter expert (understand the context and purpose of the PR) and drive best practices in clean code and architecture.
+* Only use GitHub's squash-and-merge to merge into `main`.  Else, the backport workflow can fail -- it does not handle multiple commits as with rebases.  Merges lead to an undesired merge commit.
+* Use the `backport 3` label for every PR, since main should be the image of `3` (This is not the case now, so we'll need to fix this by hand.). Add other branch labels appropriately, almost certainly `backport 2` will also be needed.
+* Close out the backport PRs in chronological order.  If you close out a PR on `main`, please close out the branch PRs as well.
```

Comment:
Close out the backport PRs in chronological order.  If you close out a PR on `main`, please close out the associated branch PRs as well.  They get generated in a few minutes by the automated backport workflow.  Once they are merged in, the backport branches can be deleted as well.